src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -326,7 +326,8 @@ private function getInsertRevisionSQL($class)
             }
 
             foreach ($class->fieldNames as $field) {
-                if (array_key_exists($field, $fields)) {
+                $columnName = $class->fieldMappings[$field]['columnName'];
```

Comment:
If you think this information is useful, you shoud use a comment. No need to clutter the variable space for that.